src/Umbraco.Web.Common/RuntimeMinification/SmidgeRuntimeMinifier.cs

```diff
@@ -169,18 +167,10 @@ public async Task<string> MinifyAsync(string fileContent, AssetType assetType)
             }
         }
 
-
         /// <inheritdoc />
-        /// <remarks>
-        /// Smidge uses the version number as cache buster (configurable).
-        /// We therefore can reset, by updating the version number in config
-        /// </remarks>
         public void Reset()
         {
-            var version = DateTime.UtcNow.Ticks.ToString();
-            _configManipulator.SaveConfigValue(Cms.Core.Constants.Configuration.ConfigRuntimeMinificationVersion, version.ToString());
+            // noop, handled by UmbracoSmidgeConfigCacheBuster.
```

Comment:
As we talked about, it would be a good idea to bring this back even though we don't use it, just so that we don't have a public method which does nothing when people would expect it to do something 😄

Reply:
Resolved 255dac0b